Tests/KlaviyoSwiftTests/StateManagementTests.swift

```diff
@@ -485,14 +485,22 @@ class StateManagementTests: XCTestCase {
     // MARK: - Test enqueue event
 
     @MainActor
-    func testEnqueueEvent() async throws {
+    func testEnqueueEvents() async throws {
```

Comment:
Updated this test to test all events and added an exception for opened push at the bottom.